rotkehlchen/exchanges/coinbase.py

```diff
@@ -220,27 +315,28 @@ def _validate_single_api_key_action(
 
         return result, ''
 
-    def validate_api_key(self) -> tuple[bool, str]:
-        """Validates that the Coinbase API key is good for usage in rotki
+    def edit_exchange_credentials(self, credentials: ExchangeAuthCredentials) -> bool:
+        changed = super().edit_exchange_credentials(credentials)
+        if credentials.api_key is not None:
+            try:
+                new_is_legacy = self.is_legacy_key(credentials.api_key)
+            except AuthenticationError as e:
+                log.error(f'Invalid API key format: {e}')
+                return False
```

Comment:
I think this is the fault of our code/docstring but `changed` here means if a new api key is given for editing. And here it is, so you should return `True`. And then this will fail here: https://github.com/rotki/rotki/blob/252891f9fcf7dbe07e4bdfa7bd261792ae203a90/rotkehlchen/exchanges/manager.py#L100 and let the user know.

Reply:
I believe I made necessary changes!